src/reference/interactive/index.pug

```diff
@@ -297,7 +297,7 @@ block reference
         img(src='/img/reference/interactive/studio/share/shareCode.png' alt='studio share code')
     h4#explicit-sharing Explicit Sharing
 
-    p The third option, “Only allow specific users to play until published”, is explicit sharing it lets you share the project with named Beam users. Only users in the list can use the project. To share a project with a user search for their username in the search box and then click add.
+    p The third option, “Only allow specific users to play until published”. Means you can explicitly share the project with named Beam users. Only users in the list can use the project. To share a project with a user search for their username in the search box and then click add.
```

Comment:
> The third option, “Only allow specific users to play until published”. Means you can explicitly share the project with named Beam users.

One super small thing - shouldn't this be `...to play until published", means you can explicitly share...`